wordle-web/components/UserNameDialog.vue

```diff
@@ -0,0 +1,28 @@
+<!-- gets the username-->
+<template>
+    <v-dialog v-model="modelValue" max-width="500" persistent>
+        <v-card>
+            <v-sheet color="primary"></v-sheet>
+            <v-card-title>Enter your user name current user is {{ userName }}</v-card-title>
```

Comment:
I would take out the "current user is _" part, since it's already displayed in the top right, and maybe add a colon. I.e "<v-card-title>Enter your username: </v-card-title>"

Reply:
```suggestion
      <v-card-title>Enter your user name</v-card-title>
        <v-card-subtitle> Curent User is {{ userName }}</v-card-subtitle>
```
I agree with this. Maybe you coule seperate it into a v-card-subtitle if you really want to display it within the dialog

Reply:
I also agree, this would be much cleaner. We'll implement this right away.